decision-records/000-template.md

```diff
@@ -2,19 +2,36 @@
 <!-- The title should be a short present tense imperative phrase, less than 50
      characters, like a git commit message. -->
 
+## Decision
+
+<!-- An overview of the decision made. -->
+
 **Date:** <!-- YYYY-MM-DD -->
 
-**Status:** <!-- proposed, accepted, rejected, deprecated, superseded -->
+## Rationale
```

Comment:
@kellylee-gds Without **Status** have we lost the ability to propose a decision, but not have it accepted yet?

Reply:
The repo has "proposals" and "decisions". I kinda figured anything that wasn't accepted yet would just be a proposal, and then when it was Set In Stone, we'd log a decision?

Reply:
Agree. I'll add **Status** back in